app/models/user_balance.rb

```diff
@@ -32,6 +32,6 @@ def self.debts_to(user)
   end
 
   def self.debtors_ids(user)
-    user.balances_as_payer.select('user_id').uniq.map(&:user_id)
+    user.balances_as_payer.select('user_id').distinct.map(&:user_id)
```

Comment:
I think it's better to do the `uniq` in the db rather than in ruby. Doesn't matter in lunchiatto's case though